seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-kudu/src/main/scala/org/apache/seatunnel/spark/source/Kudu.scala

```diff
@@ -36,7 +36,6 @@ class Kudu extends SparkBatchSource {
       "kudu.table" -> config.getString("kudu_table"))
 
     val ds = env.getSparkSession.read
-      .format("org.apache.kudu.spark.kudu")
       .options(mapConf)
       .kudu
```

Comment:
@Rianico Seems like this way is better
```scala
// Create a DataFrame that points to the Kudu table we want to query.
val df = spark.read.options(Map("kudu.master" -> "kudu.master:7051",
                                 "kudu.table" -> "default.my_table")).format("kudu").load
```

Please refer to: https://kudu.apache.org/docs/developing.html

Reply:
`format("kudu")` is a function in `org.apache.kudu:kudu-spark2_2.11`. But it only take affect from `1.9.0`, and earlier version will get an `java.lang.ClassNotFoundException: kudu.DefaultSource` Exception.   
And the version that I see in pom is `1.7.0` :
```xml
<kudu-spark.version>1.7.0</kudu-spark.version>
```
So it is better to use `kudu` function directly, we don't need to care about `kudu` or `org.apache.kudu.spark.kudu`.
  
Consider the situation if we have many spark options, the Kudu's official code example may look not so clear.

Reply:
how about upgrade the kudu-spark version? 1.7.0 seems a little old.

Reply:
> how about upgrade the kudu-spark version? 1.7.0 seems a little old.

+1

Reply:
> how about upgrade the kudu-spark version? 1.7.0 seems a little old.  

+1. How about `1.12.0` ? I have saw that many problems which belong to high priority in jira was solved in this version. 
There is also two key point about Kudu's version selection:  
- From `1.9.0`, Kudu offers a way to do junit test, we can refer to: [testing-apache-kudu-applications-on-the-jvm](https://kudu.apache.org/2019/03/19/testing-apache-kudu-applications-on-the-jvm.html#testing-apache-kudu-applications-on-the-jvm).
- From `1.10.0`, Kudu supports both full and incremental table backups via a job implemented using Apache Spark. We can expand more feature about kudu if we use the version from than `1.10.0`. But maybe we should select `1.10.1` due to [KUDU-2990](https://issues.apache.org/jira/browse/KUDU-2990).

Reply:
vote for 1.12.0

Reply:
@wuchunfu How about this idea? I'm also willing to contribute the code about the upgrade of Kudu's version and the addition of unit test. :smile: But this is another topic and maybe  I should create another issue and PR?